docs/governance/token.md

```diff
@@ -10,17 +10,19 @@ At the core of CoW Protocol lies the COW token, which serves as a governance tok
 
 #### `COW` - Main token
 
-| **Chain** | **COW Token Address** |
-| --- | --- |
-| Ethereum | [`0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB`](https://etherscan.io/token/0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB) |
+| **Chain**    | **COW Token Address**                                                                                                  |
+|--------------|------------------------------------------------------------------------------------------------------------------------|
+| Ethereum     | [`0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB`](https://etherscan.io/token/0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB)  |
 | Gnosis Chain | [`0x177127622c4a00f3d409b75571e12cb3c8973d3c`](https://gnosisscan.io/token/0x177127622c4a00f3d409b75571e12cb3c8973d3c) |
+| Arbitrum One | [`0xcb8b5cd20bdcaea9a010ac1f8d835824f5c87a04`](https://arbiscan.io/token/0xcb8b5cd20bdcaea9a010ac1f8d835824f5c87a04) |
```

Comment:
nit: I think there should be some note here that this contract is _not_ deployed / developed by CoW, but however is the bridged version of the Ethereum Mainnet canonical token.

Reply:
That would apply to the one on Gnosis Chain as well then, right?

Reply:
https://github.com/cowprotocol/docs/pull/390/commits/59c80a44cd9b36c15125dea369f02744fee9055f

Reply:
Doesn't look that good 
![image](https://github.com/cowprotocol/docs/assets/43217/62d4cfe9-ac99-42ce-8fb7-a8874d060e59)

Any tips for doing this sort of thing? Couldn't find how to do foot notes

Reply:
https://github.com/cowprotocol/docs/pull/390/commits/b2f4b2591164a0f07c1de7ec8e89817027866065